src/main/java/me/botsko/prism/utils/MaterialAliases.java

```diff
@@ -86,11 +87,11 @@ public void initMaterials(Material... materials) {
                 String matName = m.name().toLowerCase(Locale.ENGLISH);
                 String dataString;
 
-                try {
-                    dataString = BlockUtils.dataString(Bukkit.createBlockData(m));
-                } catch (IllegalArgumentException e) {
-                    continue;
-                }
+            try {
+                dataString = Utilities.dataString(Bukkit.createBlockData(m));
+            } catch (IllegalArgumentException e) {
+                continue;
```

Comment:
Codacy found an issue: ['catch' child has incorrect indentation level 16, expected level should be 20.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5612535)